spec/validators/claim/advocate_claim_validator_spec.rb

```diff
@@ -7,7 +7,14 @@
 
   let(:claim) { create(:advocate_claim) }
 
-  before { seed_fee_schemes }
+  before(:all) do
+    seed_fee_schemes
+    seed_fee_types
```

Comment:
🤔 quite a heavy operation to seed all fee type - will increase test suite runtime a fair bit

Reply:
👍 I have refactored to not seed all fee types, and use the factories instead (as per the way advocate_claim_validator _spec.rb and advocate_hardship_claim_validator_spec.rb worked before). I am also now using this approach for supplementary_claim_validator_spec.rb which previously seeded all fee types - might improve runtime??